blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -6,44 +6,47 @@
    "metadata": {},
```

Comment:
<p>I suggest import pymc as pm, because the point is to drop the numerical suffix and itll prevent people from copying this bad practice</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>